tools/rna_tools/sortmerna/tool_dependencies.xml

```diff
@@ -2,17 +2,46 @@
 <tool_dependency>
     <package name="sortmerna" version="2.0">
         <install version="1.0">
-            <actions>
-                <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
-                <action type="autoconf"/>
-                <action type="set_environment">
-                    <environment_variable name="SORTMERNADIR" action="set_to">$INSTALL_DIR</environment_variable>
-                    <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/bin</environment_variable>
-                </action>
-            </actions>
+            <actions_group>
+                <actions architecture="x86_64" os="linux">
+                        <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
+                        <action type="autoconf"/>
+                        <action type="change_directory">../</action>
+                        <action type="make_directory">$INSTALL_DIR/sortmerna-2.0/</action>
+                        <action type="move_directory_files">
+                            <source_directory>sortmerna-2.0/</source_directory>
+                            <destination_directory>$INSTALL_DIR/sortmerna-2.0/</destination_directory>
+                        </action>
+                        <action type="set_environment">
+                            <environment_variable name="SORTMERNADIR" action="set_to">$INSTALL_DIR/sortmerna-2.0/</environment_variable>
+                            <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/bin</environment_variable>
+                        </action>
+                    </actions>
+                <actions architecture="x86_64" os="darwin">
+                        <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
+                        <action type="shell_command"><![CDATA[
+                            export CC=clang && export CXX=clang++ && ./configure && make && make install
```

Comment:
@bebatut this uses Clang from LLVM, right? We do not have LLVM as dependency for Galaxy as we are using GCC. Is this working with GCC as well? Are the exports really needed? Shouldn't the system be smart enough to pick the correct compiler?

The IUC needs to decide which compiler we want to use in the future or how we can support both.